src/main/webapp/app/exercises/modeling/manage/example-modeling/example-modeling-submission.component.ts

```diff
@@ -327,7 +328,9 @@ export class ExampleModelingSubmissionComponent implements OnInit {
             return;
         }
 
-        this.totalScore = credits.reduce((a, b) => a! + b!, 0)!;
+        const maxPoints = this.exercise.maxScore! + (this.exercise.bonusPoints! ?? 0.0);
```

Comment:
Because of `bonusPoints!`, I think it's not possible for bonusPoints to be null or undefined, so no need for `?? 0.0` here

Reply:
Nice catch. Done in 3a5067d17ff9d277d3e652c08ea80adbbf1014a5